### ISSUE TITLE
Add named anchors to headlines for LBoP

### ISSUE THREAD

**Issue by rraue** [mozilla/bedrock#9645]
## Description

In https://github.com/mozilla/bedrock/issues/9523 we created the little book of privacy page. Social asked if we could add named anchors to the headlines of the page so the team can directly link to parts of the long page

- [ ] Liz to provide a list with headlines (can be just some or all headlines) and the #names for the headlines

**Comment by rraue** [comment]
Blocked by names list

**Comment by rraue** [comment]
wontfix
